docker/perf_layer_env.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Enable the performance layers by setting the evironment variables.
```

Comment:
nit: In descriptions/documentation lines like this one we describe what an entity (function, class, module) does do, not tell it what to do. Could you `s/Enable/Enables/`, `s/Run/Runs/`, `s/Test/Tests/`?